src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -260,13 +269,14 @@ const LibraryAuthoringPage = ({ returnToLibrarySelection }: LibraryAuthoringPage
               className="my-3"
             >
               <Tab eventKey={ContentType.home} title={intl.formatMessage(messages.homeTab)} />
-              <Tab eventKey={ContentType.components} title={intl.formatMessage(messages.componentsTab)} />
               <Tab eventKey={ContentType.collections} title={intl.formatMessage(messages.collectionsTab)} />
+              <Tab eventKey={ContentType.components} title={intl.formatMessage(messages.componentsTab)} />
+              <Tab eventKey={ContentType.units} title={intl.formatMessage(messages.unitsTab)} />
             </Tabs>
             <ActionRow className="my-3">
               <SearchKeywordsField className="mr-3" />
               <FilterByTags />
-              {!insideCollections && <FilterByBlockType />}
+              {!(insideCollections || insideUnits) && <FilterByBlockType />}
```

Comment:
I can get into a weird state between the Components and Units tabs:

1. In a Library, create a Component (e.g Text) and a Unit
2. From the "All Content", filter on block type = Text
3. Select the Units tab.
   Bug: shows "No matching components found in this library."
   Expected: my Unit card
2. From the "All Content", filter on block type = Unit
3. Select the Components tab.
   Bug: shows "No matching components found in this library."
   Expected: my Text card

Reply:
Thanks @pomegranited! Fixed here: 18b0fe0247865c35fa2c4702c1aeed9f565586f3